src/main/java/seedu/toluist/controller/UpdateTaskController.java

```diff
@@ -57,10 +59,17 @@ public CommandResult execute(String command) {
 
         boolean isFloating = tokens.containsKey(TaskTokenizer.TASK_FLOATING_KEYWORD);
 
+        Optional<Boolean> isHighPriority = Optional.empty();
+        if (tokens.containsKey(TaskTokenizer.TASK_PRIORITY_KEYWORD)) {
+            String taskPriority = tokens.get(TaskTokenizer.TASK_PRIORITY_KEYWORD);
+            isHighPriority = Optional.of(taskPriority.equals("high"));
```

Comment:
`cuz i thought we just need to write in user doc that we only accept "high" or "low"` then it's an error condition that has no explicit error messages